e2e/test/scenarios/filters/filter.cy.spec.js

```diff
@@ -670,21 +674,23 @@ describe("scenarios > question > filter", () => {
 
   it("should allow choosing a suggestion with Tab", () => {
     H.openOrdersTable({ mode: "notebook" });
+
     H.filter({ mode: "notebook" });
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Custom Expression").click();
+    H.popover().findByText("Custom Expression").click();
 
     // Try to auto-complete Tax
-    cy.get(".ace_text-input").type("Ta");
+    H.CustomExpressionEditor.focus().type("Ta");
 
-    // Suggestion popover shows up and this select the first one ([Created At])
-    cy.realPress("Tab");
+    H.CustomExpressionEditor.completions().should("be.visible");
+
+    // Suggestion popover shows up and this select the first one ([Tax])
+    cy.realPress("Enter");
 
     // Focus remains on the expression editor
-    cy.focused().should("have.attr", "class").and("eq", "ace_text-input");
+    cy.focused().should("have.attr", "class").and("eq", "cm-content");
 
     // Finish to complete a valid expression, i.e. [Tax] > 42
-    cy.get(".ace_text-input").type("> 42");
+    H.CustomExpressionEditor.type(" > 42");
```

Comment:
If it doesn't work without the space, then we have a regression.

Reply:
Both work!